web-dapp/test/server/is_address_confirmed.spec.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const [wallet] = require('./_utils/mocks').wallets;
+const contract = require('../../server-config').contract;
+const isAddressConfirmed = require('../../server-lib/is_address_confirmed');
+
+describe('Is Address Confirmed', () => {
+    it('should invoke the corresponding contract methods with the given wallet and address index', () => {
+        const opts = {
+            wallet,
+            addressIndex: 0,
+        };
+        let spy = jest.spyOn(contract, 'userAddressConfirmed');
+        isAddressConfirmed(opts)
+            .catch(() => {
```

Comment:
Using `catch()` because the call is going to _alway fail_ as there are no confirmed addresses.